apps/zui/src/util/error-to-string.ts

```diff
@@ -15,6 +15,9 @@ export function errorToString(e: unknown) {
     if ("error" in e && typeof e.error == "string") {
       return e.error
     } else if ("message" in e && typeof e.message === "string") {
+      if ("details" in e && Array.isArray(e.details)) {
+        return [e.message, ...e.details].join("\n")
```

Comment:
@jameskerr: react-hot-toast ignores these newlines (as well as any embedded in the strings in `e.details`) so these notifications aren't always as readable as they could be, as you can see here. Got any ideas for how we can improve on this?

<img width="1362" alt="image" src="https://github.com/brimdata/zui/assets/2574448/cb49db63-f9b8-4fde-bfd7-eabe81d6e1d5">